tsl/test/expected/bgw_policy.out

```diff
@@ -516,7 +516,7 @@ select set_integer_now_func('test_overflow_smallint', 'overflow_now');
 select add_retention_policy('test_overflow_smallint', -2) as drop_chunks_job_id \gset
 \set ON_ERROR_STOP 0
 CALL run_job(:drop_chunks_job_id);
-ERROR:  ts_interval overflow
+ERROR:  subtract integer from now smallint out of range
```

Comment:
In line with my previous comment on this error, I think this illustrates that this error doesn't make much sense in this context. What does "subtract integer from now" mean in this context. There is no connection to `run_job`.

Also, I think something is wrong here. Why is this error emitted when I try to run the job as opposed to adding the policy? There is nothing in `run_job` that explains where this error comes from since there was never any smallint or config given as argument to the function. This validation of the config should have happened when the policy was added, not when run.

Reply:
We cannot validate this when the policy is added because now() value is based on either the current timestamp or the now function: i.e. the lag for the retention policy is a dynamic value computed at job exec time. 
 Also, the user is not expected to manually run the job via run_job api. The job would be run by the bgw framework.